docs/pages/blog/base-ui-2024-plans.md

```diff
@@ -56,10 +56,11 @@ Additionally, since the `slots` and the corresponding `slotProps` are not relate
 To address these issues, we're considering adopting a new API that would assign a discrete subcomponent to each DOM node—the pattern favored by many other headless component libraries (think: `<Slider.Track />`, `<Slider.Thumb />`, etc.).
 This pattern has the potential to radically improve the customization experience, both for styles and structure.
 
-We're still fleshing out the details and plan to release an RFC on GitHub in the coming days.
-We'll update this blog post with the link when it's available.
+We're still fleshing out the details.
 We know that a significant number of projects depend on the existing API, and we want to assure you that one of our top priorities is to provide a smooth migration experience.
 
+If you have any questions or remarks about this change, feel free to comment on the [RFC](https://github.com/mui/material-ui/discussions/41085) we recently published.
```

Comment:
```suggestion
If you have any questions or remarks about this change, chime in on [the recently published RFC](https://github.com/mui/material-ui/discussions/41085).
```

@samuelsycamore, how about this? Just an alternative. And a small pet peeve of mine is when links are on small words (such as "RFC" in this case) 😅 so maybe it's easier to click on it if we extend to the full mention?